authapi/serializers.py

```diff
@@ -51,11 +51,46 @@ class UserSerializer(serializers.ModelSerializer):
         many=True, source='seedteam_set', read_only=True)
     organizations = OrganizationSummarySerializer(
         many=True, source='seedorganization_set', read_only=True)
-    email = serializers.EmailField(source='username')
-    admin = serializers.BooleanField(source='is_superuser')
+    email = serializers.EmailField()
+    admin = serializers.BooleanField(source='is_superuser', required=False)
```

Comment:
I just liked the idea of easily POSTing `{'username': '..', 'password': '..'}` to create a user, but no real objections. The other field we would have to think about being required is `active`, but it seems weird to have to specify that when creating a user. But with it having a default value, it is very easy to activate a deactivated user when updating the user details (although the reason you're updating the details is probably because you want to activate them).